desktop/packages/mullvad-vpn/test/e2e/installed/state-dependent/tunnel-state.spec.ts

```diff
@@ -16,117 +16,183 @@ const exec = promisify(execAsync);
 // IN_IP: In ip of the relay passed in `HOSTNAME`
 // CONNECTION_CHECK_URL: Url to the connection check
 
+const HOSTNAME = process.env.HOSTNAME;
+const IN_IP = process.env.IN_IP;
+const CONNECTION_CHECK_URL = process.env.CONNECTION_CHECK_URL;
```

Comment:
What about simply extracting these values in one go:

```suggestion
const {
  CONNECTION_CHECK_URL,
  HOSTNAME,
  IN_IP,
} = process.env;
```

Reply:
Thanks for the suggestion! Done.